app/helpers/spree/admin/orders_helper.rb

```diff
@@ -25,18 +25,20 @@ def line_item_shipment_price(line_item, quantity)
         Spree::Money.new(line_item.price * quantity, currency: line_item.currency)
       end
 
-      def order_links(order)
-        @order ||= order
+      def order_links(current_order)
+        @order ||= current_order
         links = []
         links << edit_order_link unless action_name == "edit"
-        links.concat(complete_order_links) if @order.complete? || @order.resumed?
-        links << ship_order_link if @order.ready_to_ship?
-        links << cancel_order_link if @order.can_cancel?
+        links.concat(complete_order_links) if order.complete? || order.resumed?
+        links << ship_order_link if order.ready_to_ship?
+        links << cancel_order_link if order.can_cancel?
         links
       end
 
       private
 
+      attr_reader :order
+
```

Comment:
Interesting. This reader is still referring to `@order` but it's now hidden behind the `attr_reader` method. While this satisfies the cop, it doesn't address the underlying problem: we are referencing a variable outside the helper that needs to be provided by the caller but there is no contract, like a method argument, to supply that variable.

I'm not sure I want to go in this direction.

Reply:
Yes, @mkllnk - I realized this later that even though this fixes the Rubocop error, the main code quality issue this cop fixes still remains. That's why I updated the fix to have an `order` passed as a parameter to each helper here:
https://github.com/openfoodfoundation/openfoodnetwork/pull/12324/commits/05f373f541e790906f70f6897b63530c0ad2fa62

I wanted to squash these both commits but there was another (_irrelevant_) commit in between these twos, so couldn't squash them without mixing the middle commit. That's why I remained it as it is. Sorry for the confusion here.

Reply:
Oh, sorry that I overlooked that. All good then.